databases/models/UserFollowUser.js

```diff
@@ -7,7 +7,7 @@ module.exports = (sequelize, DataTypes) => {
      * This method is not a part of Sequelize lifecycle.
      * The `models/index` file will call this method automatically.
      */
-    static associate(models) {
+    static associate(_models) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The parameter name in the `associate` method has been changed from `models` to `_models`, but it seems like this parameter is not being used inside the function. If it's not used, consider removing it to avoid confusion.
```diff
-    static associate(_models) {
+    static associate() {
       // define association here
     }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->